.github/workflows/release.yml

```diff
@@ -23,27 +20,27 @@ jobs:
 
       - name: Setup git env
         run: |
-          git config --global user.email "ckipp@pm.me"
-          git config --global user.name "coc-metals CI"
+          git config --global user.email "bot@scalameta.org"
+          git config --global user.name "Scalameta bot"
 
       - name: Checkout
         uses: actions/checkout@v1
 
       - name: Get the tag version
         id: get_version
-        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+        run: echo ::set-output name=NEW_NEW_VERSION::${GITHUB_REF#refs/tags/v}
```

Comment:
I was getting the `v` included in the version in the past. While it still worked, it's cleaner this way, and the same way we do it in the other extension.